lit_gpt/adapter.py

```diff
@@ -115,6 +115,11 @@ def forward(
     def from_name(cls, name: str, **kwargs: Any) -> Self:
         return cls(Config.from_name(name, **kwargs))
 
+    def _init_weights(self, module: nn.Module) -> None:
```

Comment:
Can we add a comment that this is unused but here for "backward compatibility" as you explained offline? Calling this method is not correct for adapter/lora etc, or at least shouldn't be necessary.

Same for the other methods where this is defined.